packages/scheduler/src/VirtualTimer.js

```diff
@@ -0,0 +1,60 @@
+/** @license MIT License (c) copyright 2010-2018 original author or authors */
+
+export default function newVirtualTimer () {
+  let nows = 0
```

Comment:
What is `nows`? When I see a name like that, I think of a list because of the plural. Wouldn’t `now` be fine?